Lib/test/test_cppext.py

```diff
@@ -59,29 +65,26 @@ def _check_build(self, std_cpp03, extension_name):
             python = os.path.join(venv_dir, 'bin', python_exe)
 
         def run_cmd(operation, cmd):
+            env = os.environ.copy()
+            env['CPYTHON_TEST_CPP_STD'] = 'c++03' if std_cpp03 else 'c++11'
+            env['CPYTHON_TEST_EXT_NAME'] = extension_name
             if verbose:
                 print('Run:', ' '.join(cmd))
-                subprocess.run(cmd, check=True)
+                subprocess.run(cmd, check=True, env=env)
             else:
                 proc = subprocess.run(cmd,
                                       stdout=subprocess.PIPE,
                                       stderr=subprocess.STDOUT,
-                                      text=True)
+                                      text=True,
+                                      env=env)
```

Comment:
I would’ve put `env` right after `cmd`, as they are related to the process model, whereas `text` is python processing.
Very minor of course.